euclidean_gcd/CPP/euclidean_gcd.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+
+int gcd_rec(int a, int b)
+{
+    return b == 0 ? a : gcd_rec(b, a % b);
+}
+
+
+int gcd_loop(int a , int b) {
+	while (b != 0) 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpwazjr6gq/euclidean_gcd/CPP/euclidean_gcd.cpp
+++ b/tmp/tmpwazjr6gq/euclidean_gcd/CPP/euclidean_gcd.cpp
@@ -7,7 +7,7 @@
 
 
 int gcd_loop(int a , int b) {
-	while (b != 0) 
+	while (b != 0)
     {
 		int temp = b;
 		b = a % b;
```